ravem/indico_ravem/util.py

```diff
@@ -112,13 +91,17 @@ def has_access(event_vc_room, _split_re=re.compile(r'[\s,;]+')):
     event = event_vc_room.event
     current_user = session.user
 
-    # No physical room or room is not Vidyo capable
-    if not room or not room.has_equipment('Vidyo'):
+    # No physical room or room is not videoconference capable
+    # TODO: We don't have any for room equipment attribute for zoom yet
```

Comment:
Maybe let's use a room feature here like we do in other plugins. See `get_av_capable_rooms` in the `audiovisual` plugin for an example.

Reply:
Good idea